### ISSUE TITLE
"typographic" (`lb`) vs. "topographic" (`line`)

### ISSUE THREAD

**Issue by gerritbruening** [TEIC/TEI#2367]
The wording of the elements' definitions of `lb` and `line` slightly differ: 

> `<lb>` (line beginning) marks the beginning of a new (t**y**pographic) line ...

> `<line>` contains the transcription of a t**o**pographic line ...

Assuming that this is not intended, but historical, I would like to suggest the following:

1. Check whether "topographic" is appropriate in the given context.
2. If it is, use "topographic" in both definitions, since "typographic" refers to printed material only.

**Comment by hcayless** [comment]
"Typographic" is clearly incorrect. "Topographic" must be metaphorical, and maybe there's a specialist use of the term to describe writing on a surface. But it's a metaphor that's never explained that I can see. I'd favor getting rid of it, or replacing it with something clearer.

**Comment by lb42** [comment]
"Typographic" is only incorrect if you are talking about non print materials, surely. The point there is to distinguish a typographic line from e.g. a metrical line. As for "topographic" I don't see why you think it's metaphorical or unclear. The Guidelines say "Within a zone the transcription may be organized topographically in terms of lines of writing, using the line element, or in terms of further nested zones, or as a combination of the two" .  And a random online Oxford dictionary defines "topoographical" as "relating to the arrangement or accurate representation of the physical features of an area". Which seems exactly right here.  So maybe what  needs revision is the discussion of what `<lb/>` means.

**Comment by martindholmes** [comment]
I agree with @lb42; this distinction seems intentional, apt, and accurate.

**Comment by hcayless** [comment]
1) Yes, but `<lb/>` is a perfectly usable element in encoding non-print materials, so calling it "typographic" doesn't add anything. And might make people think it isn't appropriate for non-typographic texts.
2) "Topographical" features of a text implies the text is like a map (there's our metaphor) and the markup is like (e.g.) contour lines. And that's a reasonable analogy, but I'd prefer we either explain it more clearly or stop using it.

**Comment by trishaoconnor** [comment]
Action on @trishaoconnor to rephrase the desc of <lb/>  according to @gerritbruening proposal: "...marks the beginning of a topographic line". 

@sabineseifert also notes that Council needs to update the outdated prose in the third line of note and the example in `<lb>`

>  "This element is intended to be used for marking actual line breaks"

 change to 

> "This element is intended to be used for marking the line beginning"

Additionally, @sabineseifert also highlighted that the note contains the term ‘linebreaks’ which should be updated to line beginning (see related ticket [#2603](https://github.com/TEIC/TEI/issues/2603)).

Related comment for the `<line>` element Spec page: The note of `<line>` requires rewriting for extra clarification, since the note specifically states to not use `<line>` for encoding a poem yet the first example provided is of a poem being encoded using `<line>` but there is no clarification provided. 

The clarification provided for `<lb>` would be appropriate for `<line>` as well:

> This example shows typographical line breaks within metrical lines, where they occur at different places in different editions:

**Comment by trishaoconnor** [comment]
All requested updates applied, see PR #2633.

**Comment by trishaoconnor** [comment]
Closing this issue, since the requested updates were applied and the PR was successfully merged.
